library/Exceptions/NestedValidationException.php

```diff
@@ -137,11 +137,12 @@ public function getIterator()
         $knownDepths = [];
         foreach ($recursiveIteratorIterator as $childException) {
             if ($childException instanceof self
-                && $childException->getRelated()->count() > 0
-                && $childException->getRelated()->count() < 2) {
-                continue;
+                && $childException->getRelated()->count() === 1) {
+                $relatedException = iterator_to_array($childException->getRelated())[0];
+                if (! $relatedException instanceof  AttributeException) {
```

Comment:
Thanks for the suggestion! I think that's a reasonable approach. I've updated the PR accordingly.